docs/content/latest/yugabyte-cloud/cloud-basics/create-clusters.md

```diff
@@ -76,7 +76,7 @@ If you are creating a **Paid** cluster, set the following additional options:
   - **Node Level** - a minimum of 3 nodes deployed in a single availability zone with a [replication factor](../../../architecture/docdb-replication/replication/) (RF) of 3. YugabyteDB can continue to do reads and writes even in case of a node failure, but this configuration is not resilient to cloud availability zone outages. For horizontal scaling, you can scale nodes in increments of 1.
   - **Availability Zone Level** - a minimum of 3 nodes spread across multiple availability zones with a RF of 3. YugabyteDB can continue to do reads and writes even in case of a cloud availability zone failure. This configuration provides the maximum protection for a data center failure. Recommended for production deployments. For horizontal scaling, nodes are scaled in increments of 3.
 
-- **Network Access**: If you want to use dedicated VPCs for network isolation and security, contact Yugabyte Support to set up [VPC Peering](../../cloud-network/vpc-peers) with your client VPC. Note this has to be done before you create your cluster.
+- **Network Access**: If you want to use a VPC for network isolation and security, select **Deploy this cluster in a dedicated VPC**, then select the VPC. Only VPCs using the selected cloud provider are listed, and the peering connection must be active. Refer to [VPC peering](../../cloud-network/vpc-peers/).
```

Comment:
The peering connection can not be active/created, and the cluster can still be created. However, they can't connect to it till they create a peering. But again, they can create the cluster and work on creating the peering in parallel.